### ISSUE TITLE
Bug: Dashboard doesn't open

### ISSUE THREAD

**Issue by thisisartman** [0x5bfa/FluentHub#502]
### 🪟 What version of Windows is this issue present?

🪟 Windows 11

### 🔢 What version of FluentHub are you on?

0.8.60.0 | X64 | RELEASE

### 📄 Description

App dashboard will not open.

### 🪜 Steps To Reproduce

Open app.

### 🤔 Expected behavior

Dashboard should open.

### 📑 Stack Trace

<details><summary>Stack trace details</summary><div>

```
   at System.Convert.ThrowInt32OverflowException()
   at System.Convert.ToInt32(Int64 value)
   at System.Convert.ChangeType(Object value, Type conversionType, IFormatProvider provider)
   at Octokit.PocoJsonSerializerStrategy.DeserializeObject(Object value, Type type) in /_/Octokit/SimpleJson.cs:line 1437
   at Octokit.PocoJsonSerializerStrategy.DeserializeObject(Object value, Type type) in /_/Octokit/SimpleJson.cs:line 1492
   at Octokit.PocoJsonSerializerStrategy.DeserializeObject(Object value, Type type) in /_/Octokit/SimpleJson.cs:line 1492
   at Octokit.PocoJsonSerializerStrategy.DeserializeObject(Object value, Type type) in /_/Octokit/SimpleJson.cs:line 1492
   at Octokit.PocoJsonSerializerStrategy.DeserializeObject(Object value, Type type) in /_/Octokit/SimpleJson.cs:line 1519
   at Octokit.SimpleJson.DeserializeObject[T](String json, IJsonSerializerStrategy jsonSerializerStrategy) in /_/Octokit/SimpleJson.cs:line 596
   at Octokit.Internal.JsonHttpPipeline.DeserializeResponse[T](IResponse response) in /_/Octokit/Http/JsonHttpPipeline.cs:line 62
   at Octokit.Connection.Run[T](IRequest request, CancellationToken cancellationToken, Func`2 preprocessResponseBody) in /_/Octokit/Http/Connection.cs:line 696
   at Octokit.ApiConnection.GetPage[TU](Uri uri, IDictionary`2 parameters, String accepts, ApiOptions options, Func`2 preprocessResponseBody) in /_/Octokit/Http/ApiConnection.cs:line 672
   at Octokit.ApiConnection.<>c__DisplayClass19_0`1.<<GetAll>b__0>d.MoveNext() in /_/Octokit/Http/ApiConnection.cs:line 230
--- End of stack trace from previous location ---
   at Octokit.ApiPagination.GetAllPages[T](Func`1 getFirstPage, Uri uri) in /_/Octokit/Clients/ApiPagination.cs:line 23
   at FluentHub.Octokit.Queries.Users.ActivityQueries.GetAllAsync(String login)
   at FluentHub.App.ViewModels.Viewers.DashBoardViewModel.LoadHomeContentsAsync()
   at FluentHub.App.ViewModels.Viewers.DashBoardViewModel.LoadUserHomePageAsync()
```

</div></details>

### 📸 Assets

[Log20240813.log](https://github.com/user-attachments/files/16598471/Log20240813.log)
![image](https://github.com/user-attachments/assets/04713352-ff72-42a6-a2e3-7710b7e80e73)

**Comment by Lamparter** [comment]
Duplicate of #470
@thisisartman this is a known error with the MPN/Storefront version of FluentHub. This has already been patched. Meanwhile, consider building from the source as this error is fixed in the source version. Enjoy using FluentHub! 😁

**Comment by Lamparter** [comment]
Hello @thisisartman! This issue has been fixed in v0.9 which is now available in the Store 🙂
